app/serializers/concerns/discourse_solved/topic_answer_mixin.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module ::DiscourseSolved
```

Comment:
```suggestion
module DiscourseSolved
```

Nit but I don't think `::` is necessary here